index.d.ts

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+
+declare module 'react-native-side-drawer' {
+
+	interface MenuDrawerProps {
+
+		/**
+		 * Value toggling open and close of drawer
+		 * @type boolean
+		 * @memberof MenuDrawerProps
+		 */
+		open?: boolean
+
+		/**
+		 * Drawer contents
+		 * @type number
+		 * @memberof MenuDrawerProps
+		 */
+		drawerContent?: React.ReactElement
+
+		/**
+		 * Value between 0 - 100, depicting the percentage of the screen the drawer will open
+		 * @type number
+		 * @memberof MenuDrawerProps
+		 */
+		drawerPercentage?: number
+
+		/**
+		 * Value depicting the time (in ms) the menu will slide open & close
+		 * @type number
+		 * @memberof MenuDrawerProps
+		 */
+		animationTime?: number
+
+		/**
+		 * Value toggling menu overlay or push. When overlay is true, the menu will overlay the background screen. When overlay is false, the menu will push the background screen to the side
+		 * @type bool
+		 * @memberof MenuDrawerProps
+		 */
+		overlay?: bool
```

Comment:
The 'bool' type is being used anymore by TypeScript, this should be 'boolean'.

Reply:
Agree, updated to boolean

Reply:
Thank you for the fix, programmed too much C# on the same day. haha